src/odemis/gui/test/comp_overlay_test.py

```diff
@@ -489,6 +504,43 @@ def test_cryo_feature_overlay(self):
                                refproc="Standard")
         tab_mod.main.stage = stage
         tab_mod.main.focus = focus
+        tab_mod.view_posture = model.VigilantAttribute(FM_IMAGING)
+
+        class FakeMicroscope:
+            """Mock class to simulate a microscope with a stage and focus controller without running the backend"""
+
+            def __init__(self, dependencies=None):
+                self.role = "meteor"
+                self.children = model.ListVA()
+
+        def getComponent(name="", role="stage-bare"):
+            """
+            Mock function to return a component based on its name and role.
+            :return: fake component based on name and role
+            """
+            if role == "stage-bare":
+                return stage
+            elif role == "focus":
+                return focus
+            elif name == "Linked YZ":
+                return stage
+            return None
+
+        def get_scan_rotation():
+            """
+            Mock function to return a scan rotation value without running the backend.
+            :return: float scan rotation value
+            """
+            return 0.0
+
+        # Mock getMicroscope to return FakeMicroscope
+        with patch('odemis.gui.test.comp_overlay_test.getMicroscope', return_value=FakeMicroscope()), \
+                patch('odemis.acq.move.model.getComponent', side_effect=getComponent), \
+                patch('odemis.acq.move.MeteorPostureManager._get_scan_rotation', return_value=get_scan_rotation()):
```

Comment:
I'm pretty sure you could simplify it by just replacing the call to `get_scan_rotation()` by just `0.0`.